src/mmw/js/src/modeling/gwlfe/subbasin/views.js

```diff
@@ -243,11 +245,20 @@ var SourcesTableView = Marionette.ItemView.extend({
     },
 
     templateHelpers: function() {
-        var result = this.getResult();
+        var result = this.getResult(),
+            scheme = settings.get('unit_scheme');
+
         if (!result) { return; }
+
         return {
             rows: result.Loads,
             summaryRow: result.SummaryLoads,
+            areaUnit: coreUnits[scheme].AREA_L.name,
+            areaDisplayUnit: 'AREA_L',
```

Comment:
My worker is still building so I haven't had a chance to test this yet, but since you mentioned that the Area was off by a few orders of magnitude, I think this is the problem. The units are coming in from the API as `ha`, but the `AREA_L` conversion expects units in `m²`. We can either convert that here, or use the more convenient [`AREA_L_FROM_HA`](https://github.com/WikiWatershed/model-my-watershed/blob/834c40d4ca6664aefab6d45a7c87170434141a59/src/mmw/js/src/core/units.js#L16-L25) introduced in the most recent PR. Just replace `AREA_L` with `AREA_L_FROM_HA` across this and it should work.

Reply:
Thanks. I'll try that.

Reply:
That worked!